components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.zookeeper/pom.xml

```diff
@@ -59,33 +67,18 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Private-Package>
-                            org.wso2.carbon.cluster.coordinator.zookeeper.util,
-                            org.wso2.carbon.cluster.coordinator.zookeeper.util.*,
-                            org.wso2.carbon.cluster.coordinator.zookeeper,
-                            org.wso2.carbon.cluster.coordinator.zookeeper.*
-                            java.sql*,
-                            javax.sql*
+                            org.wso2.carbon.cluster.coordinator.zookeeper.util.*
                         </Private-Package>
                         <Export-Package>
-                            !org.wso2.carbon.cluster.coordinator.zookeeper.util,
-                            !org.wso2.carbon.cluster.coordinator.zookeeper.util.*
-                            org.wso2.carbon.cluster.coordinator.zookeeper,
-                            org.wso2.carbon.cluster.coordinator.zookeeper.*
+                            !org.wso2.carbon.cluster.coordinator.zookeeper.util.*,
+                            org.wso2.carbon.cluster.coordinator.zookeeper.*;version="${carbon.coordination.exp.package.version}"
                         </Export-Package>
                         <Import-Package>
-                            org.wso2.carbon.cluster.coordinator.commons.util,
-                            org.wso2.carbon.cluster.coordinator.commons.util.*,
-                            org.wso2.carbon.cluster.coordinator.commons.node,
-                            org.wso2.carbon.cluster.coordinator.commons.node.*,
-                            org.wso2.carbon.cluster.coordinator.commons.exception,
-                            org.wso2.carbon.cluster.coordinator.commons.exception.*,
-                            org.wso2.carbon.cluster.coordinator.commons.configs,
-                            org.wso2.carbon.cluster.coordinator.commons.configs.*,
-                            org.wso2.carbon.cluster.coordinator.commons,
-                            org.wso2.carbon.cluster.coordinator.commons.*,
-                            org.apache.zookeeper.*,
-                            com.google.common.*,
-                            org.apache.commons.logging.*,
+                            org.wso2.carbon.cluster.coordinator.commons.*;version="${carbon.coordination.version}",
+                            org.apache.zookeeper.*;version="${zookeeper.imp.package.version}",
+                            org.apache.curator.*;version="${apache.curator.imp.package.version}",
+                            com.google.common.*;version="[16.0,17.0)",
```

Comment:
Better to define versions in the root pom.

Reply:
Fixed in c5ceb96